Update formula

dideler opened the pull request:
- Add Linux support (so git-duet can be installed with [linuxbrew](http://linuxbrew.sh/))
- Bump version to 0.5.0
- Fix minor faults found by homebrew audit tests
  - `brew audit --new-formula git-duet`

~~Note that when I installed git-duet with this new formula, it didn't support the new `--version` option.~~ **Update:** Wrong version was released, see https://github.com/git-duet/git-duet/issues/40

dideler committed:
Add Linux support

dideler committed:
Add formula description

As recommended by homebrew's audit tests.

dideler committed:
Prefer double quoted strings

As suggested by homebrew's audit tests.

dideler committed:
Cleanup array literal

As suggested by homebrew's audit tests: `brew audit --strict git-duet`

- No spaces
- [] instead of ()

dideler committed:
Prefer array of system commands

As suggested by homebrew's audit tests.

dideler committed:
Version should appear before checksum

As suggested by homebrew's audit tests.

dideler committed:
Bump version to 0.5.0

friegger commented:
@dideler The sha, at least for the Mac version has changed, as the wrong artefacts had been published, see https://github.com/git-duet/git-duet/issues/40. The new Mac 64bit sha is `88050ceb98480a7917106180c4d81764f94db5719ad3b458b90ac7af6cee9849`. I don't know whether the Linux artefacts are affected, too. Also `0.5.0` adds git-duet-merge. I created a PR #3  just for the Mac version, before seeing yours. You can have a look at the changes there.

dideler committed:
Install git-duet-merge

This command was added in git-duet 0.4.0, see:
https://github.com/git-duet/git-duet/releases/tag/0.4.0

dideler committed:
Update SHAs of 0.5.0 tarballs

The initial release of 0.5.0 contained the wrong binaries, see:
https://github.com/git-duet/git-duet/issues/40

dideler committed:
Change formula's test to assert version installed

Instead of printing the help, the test is a little more useful now.
Verifies that the correct version is installed.

Test executed with `brew test git-duet`.

dideler commented:
Thanks @friegger! I've fixed the SHAs for Mac and Linux and included git-duet-merge now.

@jszwedko this PR is okay on my end to be merged.

jszwedko commented:
Thank you @dideler! Apologies for the delay, I somehow did not have github notifications enabled for this repo.
